src/networking.c

```diff
@@ -1637,6 +1637,12 @@ void resetClient(client *c) {
     if (!(c->flags & CLIENT_MULTI) && prevcmd != clientCommand)
         c->flags &= ~CLIENT_TRACKING_CACHING;
 
+    /* Some commands may contain sensitive data that should
+     * not be available in the slowlog. */
+    if ((c->flags & CLIENT_PREVENT_LOGGING) && !(c->flags & CLIENT_BLOCKED)) {
```

Comment:
This flag should be cleared and restored in `call()` just like `CLIENT_FORCE_AOF` and `CLIENT_PREVENT_PROP` flags, since `call()` would be called recursively like lua script and multi/exec.

Reply:
Bizarrely, I had a dream where I realized I made this mistake. Thanks for catching it as well.